.github/workflows/ci_misc.yml

```diff
@@ -0,0 +1,156 @@
+name: B.I.O CI misc
+
+on:
+  push:
+    branches:
+      # Push events to branches matching refs/heads/master
+      - 'master'
+      # Push events to branches matching refs/heads/release*
+      - 'release*'
+  pull_request:
+
+env:
+  SEQAN3_NO_VERSION_CHECK: 1
+  TZ: Europe/Berlin
+
+defaults:
+  run:
+    shell: bash -ex {0}
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          #- name: "Snippet gcc7"
+          #  cxx: "g++-7"
+          #  cc: "gcc-7"
+          #  build: snippet
+          #  build_type: Release
+          #  build_threads: 2
+          #  test_threads: 1 # snippets create and delete files and some separate tests create/delete the same files
+          #  cmake: 3.8.2
+          #  requires_toolchain: true
+          #  requires_ccache: true
+          #  skip_build_tests: false
+          #  skip_run_tests: false
+          #  use_include_dependencies: false
+
+          - name: "Documentation"
+            build: documentation
+            build_threads: 2
+            test_threads: 2
+            cmake: 3.8.2
+            doxygen: 1.9.2
+            requires_toolchain: false
+            requires_ccache: false
+            skip_build_tests: true
+            skip_run_tests: false
+            use_include_dependencies: false
+
+    steps:
+      - name: Checkout SeqAn3
+        uses: actions/checkout@v2
+        with:
+          path: seqan3
+          fetch-depth: 2
+          submodules: true
```

Comment:
```suggestion
          submodules: recursive
```

Reply:
This is required since we need to fetch seqan3 as dependency and seqan3's dependencies, too.